packages/openactive-integration-tests/documentation/featureJson.js

```diff
@@ -0,0 +1,43 @@
+const { z } = require('zod');
```

Comment:
I also wrote a little bit of code to ensure that my understanding of the data shape of feature.json is correct (and also, this will now raise an error if anyone breaks the shape 👍 )